infra/hoogle_server.tf

```diff
@@ -18,6 +19,27 @@ resource "google_compute_firewall" "hoogle" {
   }
 }
 
+// Required only for security audit purposes
+resource "google_compute_firewall" "hoogle-ssh" {
+  name        = "hoogle-firewall-hole"
+  network     = "${google_compute_network.hoogle.name}"
+  target_tags = ["hoogle"]
+
+  source_ranges = [
+    // ???
+    "52.73.38.90/32",
```

Comment:
@nycnewman What is this?

Reply:
Apparently that's the IP you'd get if you use the non-full-tunnel North Carolina VPN and (with a local config change) instruct your computer to route all traffic through it.